package.json

```diff
@@ -119,6 +119,7 @@
   "license": "MIT",
   "dependencies": {
     "css-line-break": "^2.1.0",
+    "less": "^4.2.0",
```

Comment:
Ouch. Adding `less` to the project is already not a good idea, but adding it to the `dependencies`...

Can you provide your PR without less?